dowhy/causal_graph.py

```diff
@@ -224,15 +224,26 @@ def get_unconfounded_observed_subgraph(self):
         observed_nodes = [node for node in self._graph.nodes() if self._graph.nodes[node]["observed"] == "yes"]
         return self._graph.subgraph(observed_nodes)
 
-    def do_surgery(self, node_names, remove_outgoing_edges=False, remove_incoming_edges=False):
+    def do_surgery(
+        self,
+        node_names,
+        remove_outgoing_edges=False,
+        remove_incoming_edges=False,
+        remove_only_direct_edges=False,
+        target_node_names=None,
```

Comment:
instead of renaming target_node_names, maybe "remove_only_direct_edges" could be renamed to "remove_edges_to_targets"?

Reply:
thanks for the suggestion. I've renamed "remove_only_direct_edges" to "remove_only_direct_edges_to_targets", since we only want to remove the direct edges. It is long but I thought it is best to err on the side of clarity here.